schema/workflow.json

```diff
@@ -49,9 +49,18 @@
           "description": "Workflow data input schema definition",
           "properties": {
             "schema": {
-              "type": "string",
-              "description": "URI of the JSON Schema used to validate the workflow data input",
-              "minLength": 1
+              "oneof":[
+                {
+                  "type": "string",
+                  "description": "URI of the JSON Schema used to validate the workflow data input",
+                  "minLength": 1
+                },
+                {
+                  "type": "object",
+                  "description": "The JSON Schema object used to validate the workflow data input",
+                  "$schema": "http://json-schema.org/draft-04/schema#"
```

Comment:
Well, yes, I guess it could. The only problem is see is validating the object, if it's user-selected. I mean, in that case, it might just as well be an invalid, improper schema object. WDYT?